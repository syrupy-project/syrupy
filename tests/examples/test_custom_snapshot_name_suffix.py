def test_snapshot_custom_snapshot_name_suffix(snapshot):
    assert "Syrupy is amazing!" == snapshot(name="test_is_amazing")
    assert "Syrupy is awesome!" == snapshot(name="test_is_awesome")
