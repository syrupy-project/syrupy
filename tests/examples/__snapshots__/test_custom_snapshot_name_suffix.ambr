# name: test_snapshot_custom_snapshot_name_suffix[test_is_amazing]
  'Syrupy is amazing!'
---
# name: test_snapshot_custom_snapshot_name_suffix[test_is_awesome]
  'Syrupy is awesome!'
---
